src/microbe_stage/MicrobeStage.tscn

```diff
@@ -2091,35 +72,9 @@ __meta__ = {
 
 [node name="PauseMenu" parent="." instance=ExtResource( 11 )]
 
-[connection signal="OnOpenMenu" from="MicrobeHUD" to="PauseMenu" method="Open"]
-[connection signal="OnOpenMenuToHelp" from="MicrobeHUD" to="PauseMenu" method="OpenToHelp"]
```

Comment:
Are these signals still correctly connected?

Reply:
Seems like they aren't. I'll try to fix this.

Reply:
I noticed this as I'm pretty sure I had to fix this exact thing in the multicellular HUD, so you can look there how I hooked this up.